### PR TITLE
Render column by column instead of row by row, performs better

### PR DESCRIPTION
**Pull request by jassmith**

### CODE REVIEW

File: src/data-grid/data-grid.tsx [hunk 1]
```diff
@@ -692,18 +700,31 @@ const DataGrid: React.FunctionComponent<Props> = p => {
...
+        let row = 0;
+        ctx.font = "13px Roboto, sans-serif";
+        for (const c of effectiveCols) {
             let y = headerHeight + translateY;
+            row = cellYOffset;
+
+            ctx.save();
+            ctx.beginPath();
+            if (c.sticky) {
+                clipX = Math.max(clipX, x + c.width);
```
Comment (invaliduser) [inline src/data-grid/data-grid.tsx:715]: nitpick but I don't see x change anywhere, and it's set to 0 on 704.

Comment (jassmith) [reply src/data-grid/data-grid.tsx:715]: god catch!

Comment (jassmith) [reply src/data-grid/data-grid.tsx:715]: Actually line 821 it gets set

Comment (invaliduser) [reply src/data-grid/data-grid.tsx:715]: mb, thought 821 was referring to (deleted) 704's x

**Pull request merged**